src/icons/file-icon/FileIcon.js

```diff
@@ -1,40 +1,39 @@
 // @flow
 import * as React from 'react';
 
-import IconFileAudio from '../file/IconFileAudio';
-import IconFileBoxNote from '../file/IconFileBoxNote';
-import IconFileCode from '../file/IconFileCode';
-import IconFileDefault from '../file/IconFileDefault';
-import IconFileDocument from '../file/IconFileDocument';
-import IconFileDwg from '../file/IconFileDwg';
-import IconFileExcelSpreadsheet from '../file/IconFileExcelSpreadsheet';
-import IconFileGoogleDocs from '../file/IconFileGoogleDocs';
-import IconFileGoogleSheets from '../file/IconFileGoogleSheets';
-import IconFileGoogleSlides from '../file/IconFileGoogleSlides';
-import IconFileIllustrator from '../file/IconFileIllustrator';
-import IconFileImage from '../file/IconFileImage';
-import IconFileIndesign from '../file/IconFileIndesign';
-import IconFileKeynote from '../file/IconFileKeynote';
-import IconFileNumbers from '../file/IconFileNumbers';
-import IconFilePages from '../file/IconFilePages';
-import IconFilePDF from '../file/IconFilePDF';
-import IconFilePhotoshop from '../file/IconFilePhotoshop';
-import IconFilePowerpointPresentation from '../file/IconFilePowerpointPresentation';
-import IconFilePresentation from '../file/IconFilePresentation';
-import IconFileSpreadsheet from '../file/IconFileSpreadsheet';
-import IconFileText from '../file/IconFileText';
-import IconFileThreeD from '../file/IconFileThreeD';
-import IconFileVector from '../file/IconFileVector';
-import IconFileVideo from '../file/IconFileVideo';
-import IconFileWordDocument from '../file/IconFileWordDocument';
-import IconFileZip from '../file/IconFileZip';
+import IconFileAudio from '../imported/Icon/IconFileAudio32';
```

Comment:
I think file icons should remain logically under "file"

Reply:
I removed the extra "Icon" path from the path it deposits the assets, but didn't move where it goes since there will be non-file/-folder icons that get imported